org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManager.java

```diff
@@ -315,7 +316,7 @@ public IProject createJavaProject(IProject project, IProgressMonitor monitor) th
 		return project;
 	}
 
-	public Job updateProject(IProject project) {
```

Comment:
we might as well keep the  `update(IProject project)` flavour, to avoid the need to change to the more verbose `update(project, false)` version all over the place. Needs javadoc explaining the 2 signatures

Reply:
If we have two methods, why not call them update() and forceUpdate()? Makes more understandable without referring to javadoc. Waddayathink?